idunn/places/base.py

```diff
@@ -34,29 +33,20 @@ def __init__(self, d):
         self._wiki_resp = {}
         self.properties = {}
 
-    def get_wiki_info(self, wikidata_id, wiki_index):
-        return WikidataConnector.get_wiki_info(wikidata_id, wiki_index)
+    def get_wiki_info(self, wikidata_id, lang):
+        return wikidata_es.get_info(wikidata_id, lang)
 
     def get_wiki_index(self, lang):
-        return WikidataConnector.get_wiki_index(lang)
+        return wikidata_es.get_index(lang)
```

Comment:
This two methods `get_wiki_info` and ` get_wiki_index` could probably be removed completely.  
`get_wiki_resp` (or any better name) should be the only entrypoint to fetch the Wikipedia data associated to a POI.